ethportal-api/src/dashboard/grafana.rs

```diff
@@ -5,7 +5,7 @@ use std::fs;
 use ureq;
 
 pub const DASHBOARD_TEMPLATES: &[&str] =
-    &["./ethportal-api/src/dashboard/collected-metrics-dashboard.json.template"];
+    &["../etc/grafana/dashboards/collected-metrics-dashboard.json.template"];
```

Comment:
I'm wondering if this breaks `create_dashboard()` when running it from the released ethportal-api crate, which I suppose wouldn't include this `/etc` folder.

Reply:
Well, `create_dashboard()` is used by `ethportal-api/types/cli.rs` file but I don't think the trin cli file belongs to `ethportal-api` in the first place. I can't see a use case for using the cli without trin...